src/python_testing/TC_CNET_4_22.py

```diff
@@ -0,0 +1,188 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# See https://github.com/project-chip/connectedhomeip/blob/master/docs/testing/python.md#defining-the-ci-test-arguments
+# for details about the block below.
+#
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs:
+#   run1:
+#     app: ${ALL_CLUSTERS_APP}
+#     app-args: --discriminator 1234 --KVS kvs1 --trace-to json:${TRACE_APP}.json
+#     script-args: >
+#       --storage-path admin_storage.json
+#       --commissioning-method ble-thread
+#       --discriminator 1234
+#       --passcode 20202021
+#       --thread-dataset-hex 0e0800000000000100004a0300001135060004001fffe00708fd2ebe6d94d078c00c0402a0f7f8051000112233445566778899aabbccddeeff030e4f70656e54687265616444656d6f000300000f02081111111122222222010212340410445f2b5ca6f2a93a55ce570a70efeecb
```

Comment:
I think dataset hex will be taken from a PIXIT value. At this moment we don't have CI block for running Thread tests.